locales/locale.hu.ts

```diff
@@ -5,97 +5,97 @@ export const locale: Locale = {
-    { source: 'Clear value', target: 'Tiszta érték' },
+    { source: 'choose', target: 'válasszon' },
+    { source: 'Clear value', target: 'Érték törlése' }, 
     { source: 'loading...', target: 'Betöltés...' },
     { source: 'Nothing found', target: 'Nincs találat' },
     { source: 'Search...', target: 'Keresés...' },
-    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert' },
+    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert!' },
 
     // List components
-    { source: 'Filter', target: 'Szűrő' },
-    { source: 'Filter by field', target: 'Szűrőmezőn szűrő' },
-    { source: 'Filter by...', target: 'Szűrés vlami alapján...' },
+    { source: 'Filter', target: 'Szűrés' },
+    { source: 'Filter by field', target: 'Szűrés mező alapján' },
+    { source: 'Filter by...', target: 'Szűrés valami alapján...' },
     { source: 'Back', target: 'Vissza' },
-    { source: 'Back to {0} fields', target: 'Vissza a {0} mezőkhöz' },
-    { source: 'Choose {0} from list', target: 'Válassza a {0} lehetőséget a listából' },
-    { source: '{0} fields', target: '{0} mezők' },
-    { source: '{0} relationships', target: '{0} kapcsolatok' },
+    { source: 'Back to {0} fields', target: 'Vissza a {0} mezőhöz' },
+    { source: 'Choose {0} from list', target: 'Válasszon {0} lehetőséget a listából' },
+    { source: '{0} fields', target: '{0} mező' },
+    { source: '{0} relationships', target: '{0} kapcsolat' },
     { source: 'Aggregate {0}', target: 'Összesített {0}' },
     { source: 'No fields to select', target: 'Nincsenek kiválasztandó mezők' },
     { source: 'Enter value...', target: 'Írja be az értéket ...' },
-    { source: 'Exclude', target: 'Kizárás' },
+    { source: 'Exclude', target: 'Kivétel' },
     { source: 'Actions', target: 'Akciók' },
     { source: 'Search', target: 'Keresés' },
-    { source: 'Sort 1 → 9', target: 'Válassza az 1 → 9' },
-    { source: 'Sort 9 → 1', target: 'Válassza az 9 → 1' },
-    { source: 'Sort A → Z', target: 'Rendezze A → Z' },
-    { source: 'Sort Z → A', target: 'Rendezze Z → A' },
+    { source: 'Sort 1 → 9', target: 'Rendezés 1 → 9' },
+    { source: 'Sort 9 → 1', target: 'Rendezés 9 → 1' },
+    { source: 'Sort A → Z', target: 'Rendezés A → Z' },
+    { source: 'Sort Z → A', target: 'Rendezés Z → A' },
     { source: 'Refresh automatically', target: 'Automatikus frissítés' },
     { source: 'All', target: 'Összes' },
     { source: 'Selected', target: 'Kiválasztott' },
     {
       source: 'Unfortunately, no {0} matching your query found',
-      target: 'Sajnos nem található a lekérdezésének megfelelő {0}'
+      target: 'Sajnos nem található a lekérdezésének megfelelő elem.'
     },
-    { source: 'We are fetching {0}, please, wait...', target: 'A(z) {0} lekérése folyamatban van, kérem, várjon...' },
+    { source: 'We are fetching {0}, please, wait...', target: 'A(z) adat lekérése folyamatban van, kérem, várjon...' },
```

Comment:
Why removing {0}? There should be Collection name replacing this placeholder

Reply:
Same here, this label { source: 'records', target: 'rekordok' } is actually being inserted into {0} placeholder. Btw use cases for this label are not actual anymore, but looks like there is no need to change it.